README.md

```diff
@@ -64,6 +71,24 @@ Flags:
       --version                        print the version and exit
 ```
 
+## Integration with Kubernetes with a CronJob
+
+It is possible to use [the example cronjob in
+`./doc/etcd-defrag-cronjob.yaml`](./doc/etcd-defrag-cronjob.yaml) on Kubernetes
+environments where the etcd servers are colocated with the control plane nodes.
+
+This example CronJob runs every weekday in the morning, and works by mounting
+the `/etc/kubernetes/pki/etcd` folder inside the pod, thereby permitting to
+defragment the etcd cluster inside the Kubernetes cluster itself. For more
+complex use cases you might to adapt the `--endpoints` and/or the certificates.
+
+The example CronJob is per default configured with
+`node-role.kubernetes.io/control-plane` affinity, and with the `hostNetwork:
+true` spec, so that the `etcd` server co-located on the apiserver can be
+reached directly with `127.0.0.1:2379`. If the actual IP address of the etcd
+endpoint(s) are specified instead, the CronJob can be configured with
+`hostNetwork: true`
```

Comment:
Since the etcd is configured with "hostNetwork: true", so the tool running on the same node should be configured as `hotNetwork: true` as well. Let's just keep the best practice. 
```suggestion
reached directly with `127.0.0.1:2379`.
```

Reply:
yeah I confused myself, I wanted to express that when the endpoint IP is specified the tool doesn't need `hostNetwork: true`. it's simpler to remove that sentence altogether.